### PR TITLE
Add multi-path performance analyzer content

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/references/performance-analysis.md [hunk 1]
```diff
@@ -1,18 +1,77 @@
...
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a panel on the right side of the editor. In this panel, you can perform an in-depth performance analysis of the component.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:49]: Change "panel" to "pane".

File: en/docs/references/performance-analysis.md [hunk 2]
```diff
@@ -1,18 +1,77 @@
...
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a panel on the right side of the editor. In this panel, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
+
+#### Execution paths
+
+You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** panel. The table shows the entries in ascending order of the time it takes to process a single request in each path.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:55]:
```Suggestion
You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** pane. The table shows the entries in ascending order based on the time it takes to process a single request in each path.
```

File: en/docs/references/performance-analysis.md [hunk 3]
```diff
@@ -1,18 +1,77 @@
...
+
+![Execution paths](../assets/img/references/performance-analysis/execution-paths.png){.cInlineImage-half}
+
+The Performance Analyzer tool identifies the execution paths using the service invocations performed by each of them. If an execution path does not include any service invocation, then the latency of that path is assumed to be negligible.
+
+You can select any execution path from the table. When you select an execution path, the Performance Analyzer tool will highlight the execution path on the low-code diagram. The estimated overall performance values displayed on the top banner will change based on the execution path selected.
+
+#### Performance forecast graphs
+
+The performance forecast graphs, throughput, and latency, for the selected execution path, are found below the execution path table in the **Performance Forecast** panel.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:65]: Change the word "panel" to "pane".

File: en/docs/references/performance-analysis.md [hunk 4]
```diff
@@ -1,18 +1,77 @@
...
+The Performance Analyzer tool is an AI-based tool that generates intelligent estimations of the performance of integration-based services and APIs based on historical data. The Performance Analyzer tool allows developers to obtain forecasts at development time using advanced machine learning techniques and mathematical performance models.
+
+Traditionally, performance analysis includes performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to measure performance at the application development stage, significantly reducing the time and cost spent on performance debugging.
+
+A component may have more than one execution path serving requests. The Performance Analyzer tool can analyze all the execution paths and determine the **performance-critical path**: the path that is estimated to take the longest time to process a single request.
+
+## Advantages of using the Performance Analyzer
+
 Making accurate performance estimations before releasing an application you've developed is important due to the following reasons:
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:11]: done

File: en/docs/references/performance-analysis.md [hunk 5]
```diff
@@ -1,18 +1,77 @@
...
+- The forecasts you obtain for individual connector actions help identify any bottlenecks in each execution path.
+- You can detect any performance anti-patterns and low-quality code in your component implementation that impacts the overall performance. 
+- When you create service-level agreements (SLAs) for your application, an accurate performance forecast allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
+- You can avoid missing significant performance characteristics when combining multiple applications, including external applications.
 - You can correctly assess the cost and feasibility of running the application in a production environment.
+- You can reduce the time and cost spent on debugging performance issues by eliminating or minimizing iterative load testing efforts.
+
+## Use the Performance Analyzer
+
+You can use the Choreo Performance Analyzer on the web editor in low-code mode.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:23]: Done

File: en/docs/references/performance-analysis.md [hunk 6]
```diff
@@ -1,18 +1,77 @@
...
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** button and navigate to the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real time.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:35]: Added changes

File: en/docs/references/performance-analysis.md (hunk 2 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:55]: changed;

File: en/docs/references/performance-analysis.md (hunk 3 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:65]: done

File: en/docs/references/performance-analysis.md [hunk 7]
```diff
@@ -1,18 +1,77 @@
...
+#### Performance forecast graphs
+
+The performance forecast graphs, throughput, and latency, for the selected execution path, are found below the execution path table in the **Performance Forecast** panel.
+
+![Performance graphs](../assets/img/references/performance-analysis/performance-graphs.png){.cInlineImage-half}
 
-Traditionally, the above is achieved via performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to carry out performance measurement at the application development stage, significantly reducing the time and cost spent on performance debugging.
+The throughput and latency graphs are based on the number of users accessing the component simultaneously. By analyzing these graphs carefully, you can gain a fair understanding of how well your component will scale with the increasing load.
 
 ## Assumptions
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:71]: This section is relevant to the graphs and is important to know when you are actually reading the graphs. I think it's best to leave it in that section isn't it?

File: en/docs/references/performance-analysis.md (hunk 1 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:49]: done

File: en/docs/references/performance-analysis.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/references/performance-analysis.md:71]: But wouldn't the user have already read the graphs by the time they see this? Because they have already gone through the "Use the Performance Analyzer" section?

**Review by RukshiW — DISMISSED** [review]

File: en/docs/references/performance-analysis.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:71]: No... they see the graph only when they go to see the critical path.

File: en/docs/references/performance-analysis.md [hunk 8]
```diff
@@ -1,18 +1,77 @@
...
+
+- You can identify the performance-critical path of a component at development time. 
+- The forecasts you obtain for individual connector actions help identify any bottlenecks in each execution path.
+- You can detect any performance anti-patterns and low-quality code in your component implementation that impacts the overall performance. 
+- When you create service-level agreements (SLAs) for your application, an accurate performance forecast allows you to provide valuable insights relating to the scalability of the application and the ways to optimize its performance.
+- You can avoid missing significant performance characteristics when combining multiple applications, including external applications.
 - You can correctly assess the cost and feasibility of running the application in a production environment.
+- You can reduce the time and cost spent on debugging performance issues by eliminating or minimizing iterative load testing efforts.
+
+## Use the Performance Analyzer
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:21]: Can you reword this topic?

File: en/docs/references/performance-analysis.md [hunk 9]
```diff
@@ -1,18 +1,77 @@
...
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:43]: Please remove "The Performance Analyzer tool derives these estimates based on the user count range that you have selected."

File: en/docs/references/performance-analysis.md [hunk 10]
```diff
@@ -1,18 +1,77 @@
...
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:51]: {.cInlineImage-full} part is not rendered properly in all images. Can you check?

File: en/docs/references/performance-analysis.md [hunk 11]
```diff
@@ -1,18 +1,77 @@
...
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real time.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:35]: Shall we use one of 'real-time' or 'real time'?

File: en/docs/references/performance-analysis.md [hunk 12]
```diff
@@ -1,18 +1,77 @@
...
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:49]: 'By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green.' This part has been repeated above.

File: en/docs/references/performance-analysis.md [hunk 13]
```diff
@@ -1,18 +1,77 @@
...
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
+
+#### Execution paths
+
+You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** pane. The table shows the entries in ascending order of the time it takes to process a single request in each path.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:55]: It should be descending order right?

File: en/docs/references/performance-analysis.md (hunk 8 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:21]: what would be the message you would want it to give?

File: en/docs/references/performance-analysis.md (hunk 9 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:43]: isn't it so?

File: en/docs/references/performance-analysis.md [hunk 14]
```diff
@@ -1,18 +1,77 @@
...
+
+![Execution paths](../assets/img/references/performance-analysis/execution-paths.png){.cInlineImage-half}
+
+The Performance Analyzer tool identifies the execution paths using the service invocations performed by each of them. If an execution path does not include any service invocation, then the latency of that path is assumed to be negligible.
+
+You can select any execution path from the table. When you select an execution path, the Performance Analyzer tool will highlight the execution path on the low-code diagram. The estimated overall performance values displayed on the top banner will change based on the execution path selected.
+
+#### Performance forecast graphs
+
+The performance forecast graphs, throughput, and latency, for the selected execution path, are found below the execution path table in the **Performance Forecast** pane.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:65]: The performance forecast graphs for the selected execution path, are found below the execution path table in the **Performance Forecast** pane. These graphs show how the latency and throughput vary with the number of users accessing the component simultaneously.

File: en/docs/references/performance-analysis.md (hunk 12 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:49]: Yes it is done intentionally, as there can be users who skim through the doc without following it end to end. Since this section focuses on a sub topic, and because the starting point is to reveal the graph, mentioned it here.

File: en/docs/references/performance-analysis.md [hunk 15]
```diff
@@ -1,18 +1,77 @@
...
+
+![Performance graphs](../assets/img/references/performance-analysis/performance-graphs.png){.cInlineImage-half}
 
-Traditionally, the above is achieved via performance debugging which involves measuring the performance of services by load testing them. The Performance Analyzer allows you to carry out performance measurement at the application development stage, significantly reducing the time and cost spent on performance debugging.
+The throughput and latency graphs are based on the number of users accessing the component simultaneously. By analyzing these graphs carefully, you can gain a fair understanding of how well your component will scale with the increasing load.
```
Comment (oshadajay) [inline en/docs/references/performance-analysis.md:69]: By analyzing these graphs carefully, you can gain a fair understanding of how well your component will scale with the increasing load.

File: en/docs/references/performance-analysis.md (hunk 10 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:51]: It is rendered in the live version. GitHub does not render the CSS.

File: en/docs/references/performance-analysis.md (hunk 11 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:35]: Yup, we can do that,  we have used real-time through out the documentation. Will fix this instance.

File: en/docs/references/performance-analysis.md (hunk 13 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:55]: Fixed.

File: en/docs/references/performance-analysis.md (hunk 10 above)
Comment (oshadajay) [reply en/docs/references/performance-analysis.md:51]: Ack

File: en/docs/references/performance-analysis.md (hunk 9 above)
Comment (oshadajay) [reply en/docs/references/performance-analysis.md:43]: This gives a wrong idea that users can select the user count range.

Comment (oshadajay) [reply en/docs/references/performance-analysis.md:43]: You can add 'for each api invocation for the selected user count range' if needed.

File: en/docs/references/performance-analysis.md (hunk 8 above)
Comment (oshadajay) [reply en/docs/references/performance-analysis.md:21]: 'How to use' or 'A guide to use' sounds better in my opinion.

File: en/docs/references/performance-analysis.md (hunk 12 above)
Comment (oshadajay) [reply en/docs/references/performance-analysis.md:49]: Yet repeating the same words twice is not good for a reader who goes through the whole document.

File: en/docs/references/performance-analysis.md (hunk 8 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:21]: We use a style in our headings which we start with a verb.

File: en/docs/references/performance-analysis.md (hunk 9 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:43]: ah yes.

File: en/docs/references/performance-analysis.md (hunk 14 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:65]: reworded as The performance forecast graphs for the selected execution path are below the execution path table in the **Performance Forecast** pane. These performance forecast graphs: the latency graph and the throughput graph, changes according to the number of users accessing the component simultaneously.

File: en/docs/references/performance-analysis.md (hunk 15 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:69]: changed.

File: en/docs/references/performance-analysis.md (hunk 13 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:55]: changed

File: en/docs/references/performance-analysis.md [hunk 16]
```diff
@@ -1,18 +1,77 @@
...
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real-time. 
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
```
Comment (nadheesh) [inline en/docs/references/performance-analysis.md:37]: `{.cInlineImage-threeQuarter}` is not complied properly in the preview for each figure

File: en/docs/references/performance-analysis.md [hunk 17]
```diff
@@ -1,18 +1,77 @@
...
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
```
Comment (nadheesh) [inline en/docs/references/performance-analysis.md:39]: isn't this repeated in line 49 alos?

File: en/docs/references/performance-analysis.md (hunk 16 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:37]: Yes. GitHub doesn't render CSS. It will be applied in the live version.

File: en/docs/references/performance-analysis.md [hunk 18]
```diff
@@ -1,18 +1,77 @@
...
+
+![Execution paths](../assets/img/references/performance-analysis/execution-paths.png){.cInlineImage-half}
+
+The Performance Analyzer tool identifies the execution paths using the service invocations performed by each of them. If an execution path does not include any service invocation, then the latency of that path is assumed to be negligible.
+
+You can select any execution path from the table. When you select an execution path, the Performance Analyzer tool will highlight the execution path on the low-code diagram. The estimated overall performance values displayed on the top banner will change based on the execution path selected.
+
+#### Performance forecast graphs
+
+The performance forecast graphs for the selected execution path are below the execution path table in the **Performance Forecast** pane. These performance forecast graphs: the latency graph and the throughput graph, changes according to the number of users accessing the component simultaneously.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md]: reworded

File: en/docs/references/performance-analysis.md [hunk 19]
```diff
@@ -1,18 +1,77 @@
...
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
+
+#### Execution paths
+
+You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** pane. The table shows the entries in descending order of the time it takes to process a single request in each path.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:55]:
```suggestion
You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** pane. The table shows the entries in descending order based on the time it takes to process a single request in each path.
```

File: en/docs/references/performance-analysis.md [hunk 20]
```diff
@@ -1,18 +1,77 @@
...
+
+You can use the Choreo Performance Analyzer on the Web Editor in low-code mode. 
+
+### Analyze the performance-critical path in real-time
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:31]:
```suggestion
3. Click **Edit Code** to access the Web Editor.
```

File: en/docs/references/performance-analysis.md [hunk 21]
```diff
@@ -1,18 +1,77 @@
...
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that is indicated on the low-code editor, as a bar with the summarized overall performance estimates for each resource function.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:43]: When we are referring to the tool, let's have "Performance Analyzer" in title caps.

File: en/docs/references/performance-analysis.md (hunk 17 above)
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:39]: There is an extra space after "Click"

**Review by oshadajay — APPROVED** [review]

File: en/docs/references/performance-analysis.md [hunk 22]
```diff
@@ -1,18 +1,77 @@
...
+
+To view the real-time analysis of the performance-critical path of your component, follow the steps below:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev](https://console.choreo.dev). 
+2. Navigate to your component and click **Overview** from the left pane. 
+3. Click the **Edit Code** to access the Web Editor.
+
+#### Overall performance of the performance-critical path
+
+On the low-code editor, you will see a bar with the summarized overall performance estimates for each resource function. By default, this banner displays the forecasted latency and throughput for a selected range of user count for the performance-critical path. The upper bound of the user count range is set based on the availability of historical data. Since the Performance Analyzer tool derives these metrics in real-time,  any changes to the component will update the metrics in real-time.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:35]: Change "selected range of user count" to "selected range of user counts" - it'll be incorrect I guess.. 
reworded

File: en/docs/references/performance-analysis.md [hunk 23]
```diff
@@ -1,18 +1,77 @@
...
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that is indicated on the low-code editor, as a bar with the summarized overall performance estimates for each resource function.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/references/performance-analysis.md:49]: done

File: en/docs/references/performance-analysis.md (hunk 19 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:55]: done

File: en/docs/references/performance-analysis.md (hunk 20 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:31]: done

File: en/docs/references/performance-analysis.md (hunk 21 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:43]: done

File: en/docs/references/performance-analysis.md (hunk 17 above)
Comment (shaniR) [reply en/docs/references/performance-analysis.md:39]: done

File: en/docs/references/performance-analysis.md [hunk 24]
```diff
@@ -1,18 +1,77 @@
...
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The OPerformance Analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that is indicated on the low-code editor, as a bar with the summarized overall performance estimates for each resource function.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:43]: There is typo (OPerformance)

Comment (shaniR) [reply en/docs/references/performance-analysis.md]: fixed

File: en/docs/references/performance-analysis.md [hunk 25]
```diff
@@ -1,18 +1,77 @@
...
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The Performance Analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that is indicated on the low-code editor, as a bar with the summarized overall performance estimates for each resource function.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click **Reveal performance-critical path** on the top banner to highlight it in green. A opens a pane opens on the right side of the editor. You can perform an in-depth performance analysis of the component in this pane.
```
Comment (RukshiW) [inline en/docs/references/performance-analysis.md:49]: Change the third sentence of this paragraph to address the following

1. To correct "A opens a pane opens" to "A pane opens"
2. Shall we say "A pane opens on the right of the Web Editor." This may be a minor concern, but when you say the "right side", it can be interpreted as the "correct side", which may create an ambiguity  and make the reader wonder which is the "wrong side".

Comment (shaniR) [reply en/docs/references/performance-analysis.md:49]: done

Comment (RukshiW) [reply en/docs/references/performance-analysis.md:49]: Is this corrected? It still appears the same.

**Review by RukshiW — APPROVED** [review]
